Documentation/ReleaseNotes/5.0md

```diff
@@ -0,0 +1,317 @@
+      DOC: Document generation of MD5SUMS and SHA512SUMS files for releases
+      DOC: Add More Information section to CONTRIBUTING
+      DOC: Add ITK 5.0 RC 2 release notes
+      BUG: PythonFindEmptyClasses test for windowed-sinc functions
+      BUG: Prevent ITKMesh README.md from installation as header
+      BUG: Fix identification of Clang major version
+      COMP: ImageMaskSpatialObjectGTest array initialization
+      COMP: ImageMaskSpatialObjectGTest add braces for initialization
+      ENH: Expand ResampleImageFilter requested region by interpolator domain
+      STYLE: Use typename instead of class in ImageSink
+      STYLE: override methods should not be marked virtual
+      BUG: ImageSink defines UpdateLargestPossibleRegion
+      STYLE: StreamingProcessingObject passing void with no arguments
+      BUG: {MinimumMaximum,StatisticsImageFilter} set PrimaryOutputName
+      BUG: Remove MinimumMaximumImageFilter from module2module test pipeline
+      BUG: InterpolateImageFunction::GetRadius hidden in ITKV4_COMPATIBILITY
+      BUG: Condition Eigen export code on ITK_BINARY_DIR
+      BUG: Fix ITKv3 registration test names for property setting
+      BUG: Remove CTEST_TEST_TIME debug message
+
+MetaIO Maintainers (5):
+      MetaIO 2019-05-03 (7e16abe1)
+      MetaIO 2019-05-08 (dd5089a2)
+      MetaIO 2019-05-15 (bffb2ef1)
+      MetaIO 2019-05-16 (5010ab63)
+      MetaIO 2019-05-21 (368213e5)
+
+Niels Dekker @N-Dekker (21):
+      ENH: Add std style reverse iterators to FixedArray
+      ENH: Add ImageMaskSpatialObject::ComputeMyBoundingBoxInIndexSpace()
+      STYLE: SpatialObject() = default, const BoundingBox, Transform members
+      ENH: Added tests for ImageMaskSpatialObject::IsInside
+      ENH: Test ImageMaskSpatialObject::IsInside independent of distant pixels
+      BUG: Fix issue #785 removing BoundingBox check from ImageMask IsInside
+      PERF: Overload SpatialObject IsInsideInObjectSpace without name argument
+      BUG: Half a pixel margin ImageMaskSpatialObject::ComputeMyBoundingBox()
+      STYLE: Deprecated FixedArray member functions rBegin() and rEnd()
+      STYLE: Replace std::swap_ranges by C++11 std::swap of m_InternalArray
+      DOC: Use the term "object space" for ObjectSpace based member functions
+      STYLE: Replace unsigned short by unsigned int as FixedArray index type
+      STYLE: Explicitly defaulted default-constructors Vector Accessor classes
+      STYLE: Ensure that ContinuousIndex TCoordRep is a floating point type
+      ENH: Convenience overloads for ImageBase Transform member functions
+      ENH: Add NumberOfCorners and ComputeCorners() to BoundingBox
+      STYLE: Remove (deprecate) BoundingBox::GetCorners() + m_CornersContainer
+      COMP: Fixed GCC4 missing-field-initializers warning on `{}` in GTest
+      DOC: Fixed Doxygen description BoundingBox Get member functions
+      DOC: Fixed Doxygen description BoundingBox Get member functions
```

Comment:
Note that some commits appear twice, in this list. No show stopper, of course. It appears that such a duplicated commit actually has two different SHA's, in git! Beyond my git understanding!

Reply:
@N-Dekker I imagine some comments were merged into master, then rebased onto the release branch then merged again into the release branch. Best practice should be to base these commits off of the release branch to begin with then merge the same commit (same hash) into master and again merge into release branch.

Reply:
@blowekamp Thanks for your suggestion. I think something like this has happened: First, my contribution "DOC: Fixed Doxygen..." got onto the master branch, with the merge of my pull request #916 Then Matt @thewtex possibly did a `git rebase master` of his branch (his fork), for his own pull request #934, which brought my contribution from the master into his branch. Eventually his pull request #934 was also merged onto the master, thereby adding my contribution for the second time!

Disclaimer: I'm not 100% sure... git keeps surprising me  :confused:

Reply:
@N-Dekker To explain: when we backport commits, for example, that were branched from `master` but are cherry-picked onto `release`, they are a new commit in Git-land because they have a different parent commit. On the one hand, this does make sense because the same content added at different times in the history could have a different result. On the other hand, the duplicate entries can be confusing. This can result in the apparent duplicate entry as @blowekamp mentioned.